src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigToJobMapper.java

```diff
@@ -63,7 +64,9 @@ public static FlowDefinition mapBuildConfigToFlow(BuildConfig bc) throws IOExcep
 		BuildSource source = null;
 		String jenkinsfile = null;
 		String jenkinsfilePath = null;
+    GitSCM gitSCM = null;
```

Comment:
this variable is not used anywhere else than in https://github.com/openshift/jenkins-sync-plugin/pull/360/files#diff-fbc124cdb4f21738ce080e4247a0a6fbR89 

We should declare it only there.